app/config/locale/translations/ta.json

```diff
@@ -3,30 +3,30 @@
     "settings.locale": "ta",
     "settings.direction": "ltr",
     "emails.sender": "%s குழு",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "கணக்கு சரிபார்ப்பு",
+    "emails.verification.hello": "நமஸ்தே {{name}}",
+    "emails.verification.body": "உங்கள் மின்னஞ்சல் முகவரியை சரிபார்க்க இந்த இணைப்பைப் பின்தொடரவும்.",
+    "emails.verification.footer": "இந்த முகவரியை சரிபார்க்க நீங்கள் கேட்கவில்லை என்றால், இந்த செய்தியை நீங்கள் புறக்கணிக்கலாம்.",
```

Comment:
'.....இந்த செய்தியை நீங்கள் புறக்கணிக்கலாம்' instead of this, you can say '.....இந்த செய்தியை நீங்கள் தவிர்க்கலாம்'